src/data/h1/tags/scenario.yml

```diff
@@ -728,37 +728,79 @@ comments:
         - name: recorded animation event stream
           en: ...
     - name: netgame flags
-      en: ...
+      en: Multiplayer-specific markers like flag locations, oddball spawns, and teleporters.
       fields:
         - name: position
-          en: ...
+          en: The location in 3D space of this netgame flag.
         - name: facing
-          en: ...
+          en: >
+            The direction that the netgame flag faces (yaw). How this is used
+            depends on the _type_ of the flag.
         - name: type
-          en: ...
+          en: Determines which purpose this netgame flag serves.
           options:
             - name: ctf flag
-              en: ...
+              en: >
+                **The CTF flags MUST be the first two entries in the netgame flags list**,
+                or players will be stuck "waiting for space to clear" instead of spawning
+                or the game may crash. It does not matter which team index is placed first,
+                just that the first two entries are the flags. Red team's flag uses team
+                index `0`, and blue team `1`. The _facing_ value is not used to orient
```

Comment:
Do we know what the `facing` value is used for? Or is it just unused for this flag type?

Reply:
Unused for this flag type, but is used for teleporter nodes.